dashboard/app/models/course_offering.rb

```diff
@@ -109,6 +109,11 @@ def path_to_latest_published_version(locale_code = 'en-us')
     latest_published_version(locale_code).content_root.link
   end
 
+  def display_name_with_year(locale_code = 'en-us')
```

Comment:
can this be renamed to `display_name_with_latest_year` or something? This name buries the fact that it will always return the year of the latest version to me. Can definitely be done as a follow up though

Reply:
Good call -- will do that now!